### PR TITLE
Feature/zip 325 update tasklist page full screen mode

### PR DESCRIPTION
**Pull request by zubairaltaf10**
https://github.com/Unzipped-Technologies/unzipped/assets/52136365/fcbd4a48-81c2-4798-a53f-925817584404

### CODE REVIEW

File: components/unzipped/dashboard/Kanban/AssignedToList.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+        },
+    },
+}));
+
+
+const USERS_ARR = [
+    {
+        userId: 1,
+        firstName: "Aalishan ",
+        lastName: "Jami"
```
Comment (Alien4Hire) [inline components/unzipped/dashboard/Kanban/AssignedToList.js:54]: Looks like we are hard coding the user's here. The users that can be selected should be the owner of the project and any freelancers hired for the project. See how it works in the normal task list page.

File: components/unzipped/dashboard/Kanban/MyProjectsLists.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        },
+    },
+    '& .MuiModal-root-MuiPopover-root-MuiMenu-root': { left: "175px" }
+}));
+
+
+
+const projects = [
+    {
+        name: 'Unzipped',
```
Comment (Alien4Hire) [inline components/unzipped/dashboard/Kanban/MyProjectsLists.js:85]: Projects are hard coded here. They should come from the logged in user's projects that they have started, or that they have an open contract for.

File: components/unzipped/dashboard/Kanban/ProjectusersDropdown.js [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+            padding: '2px 0',
+        },
+    },
+}));
+
+
+const USERS_ARR = [
+    {
+        email: "jasonmaynoard@gmail.com",
+        firstName: "Jason",
```
Comment (Alien4Hire) [inline components/unzipped/dashboard/Kanban/ProjectusersDropdown.js:45]: Again, the users here should be the owner of the project and any employees that have an open contract with the project. This way any employees can be assigned to tickets, or the ticket can be unassigned, or the project owner can assign themselves to the ticket.

File: components/unzipped/dashboard/Kanban/ProjectusersDropdown.js [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+                        <List sx={{
+                            width: '100%',
+                            maxWidth: 360,
+                            bgcolor: 'background.paper',
+                            margin: 0
+                        }}>
+                            <ListItem alignItems="flex-start" sx={{ padding: '0 !important' }}>
+                                {renderTextContainer("AN")}
+                                <ListItemText
+                                    primary={'Jason Maynard'}
```
Comment (Alien4Hire) [inline components/unzipped/dashboard/Kanban/ProjectusersDropdown.js:132]: We should not hard code this kind of data anywhere in the site.

File: components/unzipped/dashboard/Kanban/KanbanBoard.js [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+  }
+];
+
+const columnsFromBackend = {
+  [uuid()]: {
+    name: "New",
+    items: itemsFromBackend
+  },
+  [uuid()]: {
+    name: "To do",
```
Comment (Alien4Hire) [inline components/unzipped/dashboard/Kanban/KanbanBoard.js:162]: These items (in progress, done, etc.) Should not be hard coded. They are the tags that are associated with the project. This way a project owner can create and remove columns.

File: components/unzipped/dashboard/Kanban/KanbanBoard.js [hunk 6]
```diff
@@ -0,0 +1,327 @@
...
+  width: 100%;
+  min-height: ${(props) => props.minHeight}px; /* Dynamic height */
+  border-radius: 8px;
+`;
+
+// Data and Functions
+const itemsFromBackend = [
+  {
+    id: uuid(),
+    ticketNumber: 101,
```
Comment (Alien4Hire) [inline components/unzipped/dashboard/Kanban/KanbanBoard.js:117]: Tickets should be pulled from tasks that are associated with the selected project and department, and should be displayed in the column that corresponds with their selected tag.

**Review by Alien4Hire — CHANGES_REQUESTED** [review]

File: components/unzipped/dashboard/Kanban/KanbanBoard.js (hunk 6 above)
Comment (zubairaltaf10) [reply components/unzipped/dashboard/Kanban/KanbanBoard.js:117]: @Alien4Hire 
Initially, there was a hard-codeded data there. I have continued  with that approach. Currently we're only working with the ui component to make it functional and working features of drag and drop. I'll have these feature get implemented in the current pull request or in the new ticket ?

Comment (Alien4Hire) [reply components/unzipped/dashboard/Kanban/KanbanBoard.js:117]: I'll go ahead and merge this, then we can do it in a new pr

**Pull request merged**